deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/init

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
For future archeologists, do you mind adding a quick comment describing why this file exists, and what it's intent is?

Reply:
future archeologists hahaha ! love it ! hello future archeologists from 2019 !

Reply:
Explaining init(1) ?

Or maybe what is the difference between this init and the normal one would be more appropriate. It doesn’t really help that the final `/sbin/init` is actually **not** init but just Lennart (systemd) posing

Reply:
I for one didn't realize this took the place of the system init, though it makes sense given the location. 

It'd be nice to have an explanation of why data is being copied from one directory to another before exec'ing out to another init. =)

Reply:
Settled for adding a reference to `switch_root(8)`

>DESCRIPTION
>       switch_root moves already mounted /proc, /dev, /sys and /run to newroot and makes newroot the new root filesystem and starts init process.
>
> WARNING: switch_root removes recursively all files and directories on the current root filesystem.